docs/users/intro.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 1
+---
+# BobaBoard Basics
+
+_"BobaBoard will move you and fill you with a feeling of deep peace. I hope your time here will be pleasant and your memories will last forever."_
+
+## What is BobaBoard?
+**The Simple Answer:** A social media site made for fandom by people in fandom.
+
+**The Detailed Answer:** BobaBoard is the brain child of Ms. Boba who woke up to tumblr's porn ban and decided enough was enough. She set out to create an open source platform that could meet the evolving demands of fandom, a place that could support the needs of varying communities by being adaptable, stable, and fun to use, where fandom made the rules.
+
+The result is a social media site that's part-forum, part-Discord, part-LJ clone, and yet none of those at all.
+
+## How BobaBoard is Different...
+We hear you: you already have 37 social accounts and maybe use 3 of them, if you squint, why would you want another? Why give another startup platform a chance? What's _different_ about this place?
+
+### Realms
+:::info
+
+This section of the guide is still in development! We're still in the process of figuring out how Realms will work. The general idea is explained below, but your input will help shape things!
+
+:::
+
+BobaBoard is made up of distinct areas called **Realms**. Like LJ and its clones (and to some extent, Discord), each Realm is its own domain within the BobaBoard "network". Each Realm may have different goals, cultures, and rules.
+
+As BobaBoard expands to be open to the public, Moderators will be able to establish their own Realms. Users will have to join each Realm they want posting and reading access to and explicitly agree to their rules.
+
+There are currently two Realms making up BobaBoard:
+- V0: The Alpha
+  > The OG BobaBoard. A closed Alpha for Ms. Boba's friends, Boba volunteers, and early adopters.
+
+- Fandom Coders
+  > The first official "Realm". It's a community of fandom people who also code or want to learn to code, particularly if they use coding in their fandom activities!
+
+For more information on each Realm, visit their sections in the guide.
+
+### Boards, Not Blogs
+Each **Realm** is made up of multiple Boards. Boards are similar to the boards in a forum or modern Discord channels; they define the broad topics of conversations and give users a heads-up as to what they can expect to find.
+
+To interact in the Realm, you simply post to whatever you find most appropriate. Don't worry about getting it wrong: there is no wrong. If something might fit in more than one place, simply choose the one that's most appropriate according to your own nebulous criteria.
+
+_"The boards are more what you'd call guidelines than actual rules."_
+
+Unlike other fandom spaces, there's no personal "blog" or "page" on BobaBoard. There's no user address that you can give someone so that they can see all your posts. You won't have followers, friends, or mutuals.
```

Comment:
Maybe we could simply say "at this stage, there's no personal "blog" or "page" on BB".

I would stress that we're focused on creating a better _fandom community_ experience (maybe mention forums?), and that personal solutions might be incorporated in the future. It could be fun to point people to the Personal Website Club page, if we ever have one :P